.vsts-ci/linux.yml

```diff
@@ -80,23 +80,24 @@ jobs:
   - template: ../tools/releaseBuild/azureDevOps/templates/insert-nuget-config-azfeed.yml
 
   - powershell: |
-      tools/travis.ps1 -Stage Bootstrap
+      Import-Module .\tools\ci.psm1
+      Invoke-LinuxTests
     displayName: Bootstrap
     condition: succeeded()
 
   - powershell: |
       $ErrorActionPreference = 'continue'
-      tools/travis.ps1 -NugetKey $(NUGET_KEY)
+      tools/ci.psm1 -NugetKey $(NUGET_KEY)
     displayName: Build and test
     condition: succeeded()
 
   - powershell: |
-      tools/travis.ps1 -Stage Failure
+      Write-Host "Tests failed."
```

Comment:
I think you could probably just log success or fail inside of the psm1 and then you can safely take out this task.